jwql/website/apps/jwql/bokeh_utils.py

```diff
@@ -0,0 +1,32 @@
+"""Various Bokeh-related utility functions for the ``jwql`` project.
```

Comment:
I'm kind of bummed, because bokeh-related functions are split between bokeh_containers and bokeh_utils. I had to create bokeh_utils here in order to avoid a circular import that was affecting the dark monitor pages.

Reply:
It would be nice to be able to consolidate all of it into one module. From slack, it sounds like you have an idea of how to solve this issue. Do we have an open issue for this right now?

Reply:
Not yet, but I can create one. I think it would be pretty straightforward to move the code that makes the figure tabs for each monitor from bokeh_containers.py. All that would be left then is a collection of utils-type functions. Maybe at that point we can rename it to bokeh_utils or something more accurate than bokeh_containers.

Reply:
Sounds good to me. I think the `_containers` lingo comes from django. I am cool with that, let's move that discussion there once the issue is made. I will go ahead and close this conversation so we can get this in! Thanks again!